packages/api/src/controllers/stream.ts

```diff
@@ -323,7 +323,7 @@ function setRecordingStatus(
   if (session.record && session.recordObjectStoreId && session.lastSeen > 0) {
     const isReady = session.lastSeen > 0 && session.lastSeen < olderThen;
     session.recordingStatus = isReady ? "ready" : "waiting";
-    if (isReady || (req.user.admin && forceUrl)) {
```

Comment:
So I think the context is this: https://github.com/livepeer/livepeer-com/issues/949#issuecomment-1064580349

Removing that filter, we keep it a safer threshold for users to recover their sessions, while still allow other users to force getting the recording URL through that query-string. I've linked the code way too many times to users so they can copy the way we create the URLs, might as well expose the `forceUrl` option to them in case they know what they are doing.